src/onnx/parse_slice.cpp

```diff
@@ -97,6 +93,22 @@ struct parse_slice : op_parser<parse_slice>
         }
 
         auto ins = info.add_instruction(op, args[0]);
+        if(std::any_of(steps.begin(), steps.end(), [](auto s) { return std::abs(s) != 1; }))
+        {
+            std::vector<int64_t> nsteps;
+            std::transform(steps.begin(), steps.end(), std::back_inserter(nsteps), [](auto s) {
+                return std::abs(s);
+            });
+            auto lens      = args[0]->get_shape().lens().size();
+            auto axes_norm = op.axes;
+            for(auto& axis : axes_norm)
+            {
+                while(axis < 0)
+                    axis = axis + lens; // need to be positive for step
+            }
```

Comment:
We should normalize the axes in the reverse operator.

Reply:
via the normalize attribute.

Reply:
Do you mean step operator? [Reverse](https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/blob/e00479af6dc513d8f83e6da773014dee51ce1c2c/src/include/migraphx/op/reverse.hpp#L36) already normalizes axes. I had to add the line because step needed axis normalized, otherwise complained.

Reply:
Sorry, I meant the step.

Reply:
Done